shell/browser/native_window_views.cc

```diff
@@ -1637,10 +1637,21 @@ void NativeWindowViews::OnWidgetBoundsChanged(views::Widget* changed_widget,
   if (changed_widget != widget())
     return;
 
-  // Note: We intentionally use `GetBounds()` instead of `bounds` to properly
-  // handle minimized windows on Windows.
+  // |GetWindowBoundsInScreen| has a ~1 pixel margin of error, so if we check
+  // existing bounds directly against the new bounds without accounting for that
+  // we'll have constant false positives when the window is moving but the user
+  // hasn't changed its size at all.
+  auto areWithinOnePixel = [](gfx::Size old_size, gfx::Size new_size) -> bool {
+    bool within_range_width =
+        std::abs(old_size.width() - new_size.width()) <= 1;
+    bool within_range_height =
+        std::abs(old_size.height() - new_size.height()) <= 1;
+    return within_range_width && within_range_height;
```

Comment:
```suggestion
    return base::IsApproximatelyEqual(old_size.width(), new_size.width(), 1) &&
           base::IsApproximatelyEqual(old_size.height(), new_size.height(), 1);
```

(probably needs `#include "base/numerics/ranges.h"`)

Reply:
ah thanks!! I was hoping there was a base helper for this but couldn't find it :D

Reply:
LOLl apparently it made it harder to backport so :man_shrugging: one step forward, one step back